### ISSUE TITLE
Unable to Install “Synology Surveillance Station” via Package Center

### ISSUE THREAD

**Issue by kpinarci** [vdsm/virtual-dsm#819]
### Operating system

24.04.1 LTS

### Description

I am attempting to install the “Synology Surveillance Station” package through the Synology Package Center but receive the following error message:

`The package could not be installed.
Installation cannot be authorized. 
Please check your internet connection and try again.`

Despite this, I can successfully install other packages such as the Storage Analyzer and Log Center, and I have verified that my internet connection is stable and functional.

I am unsure what could be causing this specific issue with the Surveillance Station package installation.

### Docker compose

services:
  dsm:
    container_name: vdsm
    image: vdsm/virtual-dsm:7.19
    environment:
      TZ: Europe/Berlin
      DHCP: "Y"
      DISK_SIZE: "650G"
      RAM_SIZE: "5G"
      CPU_CORES: "4"
    devices:
      - /dev/kvm
      - /dev/vhost-net
    device_cgroup_rules:
      - 'c *:* rwm'
    networks:
      vdsm_net:
    dns:
      - 192.168.19.1
      - 1.1.1.1
    cap_add:
      - NET_ADMIN
    ports:
      - 5000:5000
      - 5001:5001
      - 9900:9900
      - 9901:9901
    volumes:
      - /var/vdsm/data:/storage
    stop_grace_period: 2m

networks:
  vdsm_net:
    driver: macvlan
    attachable: true
    driver_opts:
      parent: eth0
    ipam:
      config:
        - subnet: 192.168.19.0/24
          gateway: 192.168.19.1

### Docker log

302243] iSCSI:extent_pool.c:766:ep_init syno_extent_pool successfully initialized
vdsm  | [   31.335215] iSCSI:target_core_device.c:612:se_dev_align_max_sectors Rounding down aligned max_sectors from 4294967295 to 4294967288
vdsm  | [   31.352709] iSCSI:target_core_configfs.c:5763:target_init_dbroot db_root: cannot open: /etc/target
vdsm  | [   31.370992] iSCSI:target_core_lunbackup.c:366:init_io_buffer_head 2048 buffers allocated, total 8388608 bytes successfully
vdsm  | [   31.414959] iSCSI:target_core_file.c:152:fd_attach_hba RODSP plugin for fileio is enabled.
vdsm  | [   31.427863] iSCSI:target_core_file.c:159:fd_attach_hba ODX Token Manager is enabled.
vdsm  | [   31.439987] iSCSI:target_core_multi_file.c:91:fd_attach_hba RODSP plugin for multifile is enabled.
vdsm  | [   31.452864] iSCSI:target_core_ep.c:795:ep_attach_hba RODSP plugin for epio is enabled.
vdsm  | [   31.464270] iSCSI:target_core_ep.c:802:ep_attach_hba ODX Token Manager is enabled.
vdsm  | [   31.600862] workqueue: max_active 1024 requested for vhost_scsi is out of range, clamping between 1 and 512
vdsm  |
vdsm  | ❯ -----------------------------------------------------------
vdsm  | ❯  You can now login to DSM at http://192.168.19.227:5000
vdsm  | ❯ -----------------------------------------------------------
vdsm  |
vdsm  | [   34.139549] findhostd uses obsolete (PF_INET,SOCK_PACKET)
vdsm  | [   34.626415] fuse init (API version 7.23)

### Screenshots (optional)

![Bildschirmfoto 2024-09-28 um 22 37 13](https://github.com/user-attachments/assets/290c7b87-8a41-455a-b2d5-f265c644f932)

![Bildschirmfoto 2024-09-28 um 22 39 44](https://github.com/user-attachments/assets/6bc2bf00-7cf2-4787-9eec-108f5e5ace02)

**Comment by Skyfay** [comment]
The installation worked for me. There was an installation error towards the end but the application is running.
I have installed the same DSM version. I assume you have already tried a restart? 
My docker configuration is basically the same, except that I have specified the following parameters:
MAC, GUEST_SERIAL, HOST_SERIAL, HOST_MAC and HOST_MODEL

**Comment by kpinarci** [comment]
> Maybe this is something new in Virtual DSM 7.2.2? I know I never seen this before in 7.2.1 and lower.

I have installed the new `version:7.2.2-72806` and I have the same problem.

**Comment by kpinarci** [comment]
> The installation worked for me. There was an installation error towards the end but the application is running. I have installed the same DSM version. I assume you have already tried a restart? My docker configuration is basically the same, except that I have specified the following parameters: MAC, GUEST_SERIAL, HOST_SERIAL, HOST_MAC and HOST_MODEL

Yes, I’ve restarted the containers and the VM multiple times, but the issue persists. I haven’t set the variables yet. Apologies if this is a basic question, but where can I find the values for the variables like MAC, GUEST_SERIAL, HOST_SERIAL, HOST_MAC, and HOST_MODEL? For example, how do I determine the MAC or the *_SERIAL values?

**Comment by Skyfay** [comment]
Do you have an ‘official’ Synology Nas device?

**Comment by kpinarci** [comment]
> Do you have an ‘official’ Synology Nas device?

Ah, I see. I need to use the serial number and MAC address of my Synology. Is it possible to use both them in different networks?

**Comment by Skyfay** [comment]
> > Do you have an ‘official’ Synology Nas device?
> 
> Ah, I see. I need to use the serial number and MAC address of my Synology. Is it possible to use both them in different networks?

Yes its possible. You just need the Host Mac address and the Host serial key. In addition you need to create a VM in VVM (Synology Virtual Machine Manager) and get the VM details like MAC, Serial etc.

**Comment by kpinarci** [comment]
> > > Do you have an ‘official’ Synology Nas device?
> 
> > 
> 
> > Ah, I see. I need to use the serial number and MAC address of my Synology. Is it possible to use both them in different networks?
> 
> 
> 
> Yes its possible. You just need the Host Mac address and the Host serial key. In addition you need to create a VM in VVM (Synology Virtual Machine Manager) and get the VM details like MAC, Serial etc.

All right. I will try it out and report back, thank you very much.

**Comment by kpinarci** [comment]
> HOST_MODEL

After adding the following settings:

	•	HOST_MAC: 00-**
	•	HOST_MODEL: “DS216J”
	•	HOST_SERIAL: 16****
	•	GUEST_SERIAL: 16****

I was able to install the Surveillance Station, but now I cannot add a license. I receive the following error message: “Connection failed. Please check your network settings.”

**Comment by Skyfay** [comment]
Why do you want to add a license, what is the goal? Isn't it possible to add IP cameras without a license?
I don't use Surveillance Station myself, so I don't have a license with which I could test it.

**Comment by kpinarci** [comment]
> Why do you want to add a license, what is the goal? Isn't it possible to add IP cameras without a license?
> 
> I don't use Surveillance Station myself, so I don't have a license with which I could test it.
> 
> 
> 
> Btw. i would also recommend adding "MAC:" (the MAC address from the VM you created to get the GUEST_SERIAL)

Yes, exactly. I want to add cameras, and for each camera, a separate license needs to be added. As soon as I add the license, I encounter an error. Surveillance Station recognizes the cameras but cannot add them because no license is available.

Thank you for the tip. I will try using the MAC.

**Comment by Skyfay** [comment]
How are you trying to add the license? There are several as far as I have seen.

**Comment by kpinarci** [comment]
> How are you trying to add the license? There are several as far as I have seen.

Via main menu --> License --> Add
Which has actually always worked with Synology hardware.
